### PR TITLE
ACME Upgrade Docs

### PR DESCRIPTION
**Pull request by hawkowl**

### CODE REVIEW

File: changelog.d/4528.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Synapse can now automatically provision TLS certificates via ACME (the protocol used by CAs like Let's Encrypt).
```
Comment (richvdh) [inline changelog.d/4528.feature]: I fear this needs to say something different given it's going in post-RC

Comment (hawkowl) [reply changelog.d/4528.feature]: does it? i'm confused

Comment (hawkowl) [reply changelog.d/4528.feature]: oh

Comment (hawkowl) [reply changelog.d/4528.feature]: done

**Pull request merged**